src/site/_includes/components/PostCard.js

```diff
@@ -119,29 +128,52 @@ module.exports = ({post}) => {
     `;
   }
 
+  function renderChips() {
+    if (!displayedTags.length) {
+      return;
+    }
+    return html`
+      <div class="w-card__chips w-chips">
+        ${displayedTags.map((displayedTag) => {
+          return html`
+            <a class="w-chip" href="${displayedTag.href}"
+              >${displayedTag.title}</a
+            >
+          `;
+        })}
+      </div>
+    `;
+  }
+
   return html`
-    <a href="${url}" class="w-card">
+    <div class="w-card">
       <article class="w-post-card">
-        <div
-          class="w-post-card__cover ${thumbnail &&
-            `w-post-card__cover--with-image`}"
-        >
-          ${thumbnail && renderThumbnail(url, thumbnail, alt)}
-          <h2
-            class="${thumbnail
-              ? `w-post-card__headline--with-image`
-              : `w-post-card__headline`}"
+        <a class="w-card--link" href="${url}">
```

Comment:
```suggestion
        <a class="w-card__link" href="${url}">
```

__ signifies an element, -- signifies a modifier

Reply:
actually i think this should be post-card specific so something like `w-post-card__link`.